### ISSUE TITLE
Disabled select: bottom line position jumps

### ISSUE THREAD

**Issue by robcaldecott** [material-components/material-components-web#2211]
### What MDC Web Version are you using?

> 0.30.0

### What browser(s) is this bug affecting?

Chrome

### What OS are you using?

Windows 10

### What are the steps to reproduce the bug?

* Open the select demo page: https://material-components-web.appspot.com/select.html
* Scroll down to **Fully-Featured JS Component**
* Toggle **Disabled** on and off.

### What is the expected behavior?

The position of the disabled bottom line should not change,

### What is the actual behavior?

The bottom line shifts up a few pixels.

### Any other information you believe would be useful?

The "box" variant is OK.

**Comment by kfranqueiro** [comment]
Thanks for reporting this!
